### ISSUE TITLE
Panic when parsing RIB dump

### ISSUE THREAD

**Issue by mbUSC** [bgpkit/bgpkit-parser#156]
I encountered this error while parsing https://data.ris.ripe.net/rrc22/2023.04/bview.20230402.0000.gz:

```
thread '<unnamed>' panicked at /Users/student/.cargo/registry/src/index.crates.io-6f17d22bba15001f/bgpkit-parser-0.10.2/src/models/bgp/attributes/aspath.rs:627:43:
called `Option::unwrap()` on a `None` value
stack backtrace:
   0: rust_begin_unwind
             at /rustc/a165f1f65015b1bd4afd2ec50700aaacf2e0c485/library/std/src/panicking.rs:645:5
   1: core::panicking::panic_fmt
             at /rustc/a165f1f65015b1bd4afd2ec50700aaacf2e0c485/library/core/src/panicking.rs:72:14
   2: core::panicking::panic
             at /rustc/a165f1f65015b1bd4afd2ec50700aaacf2e0c485/library/core/src/panicking.rs:145:5
   3: core::option::unwrap_failed
             at /rustc/a165f1f65015b1bd4afd2ec50700aaacf2e0c485/library/core/src/option.rs:1985:5
   4: core::option::Option<T>::unwrap
             at /rustc/a165f1f65015b1bd4afd2ec50700aaacf2e0c485/library/core/src/option.rs:933:21
   5: bgpkit_parser::models::bgp::attributes::aspath::AsPath::merge_aspath_as4path
             at /Users/student/.cargo/registry/src/index.crates.io-6f17d22bba15001f/bgpkit-parser-0.10.2/src/models/bgp/attributes/aspath.rs:627:36
   6: bgpkit_parser::parser::mrt::mrt_elem::Elementor::record_to_elems
             at /Users/student/.cargo/registry/src/index.crates.io-6f17d22bba15001f/bgpkit-parser-0.10.2/src/parser/mrt/mrt_elem.rs:390:42
   7: <bgpkit_parser::parser::iters::ElemIterator<R> as core::iter::traits::iterator::Iterator>::next
             at /Users/student/.cargo/registry/src/index.crates.io-6f17d22bba15001f/bgpkit-parser-0.10.2/src/parser/iters.rs:166:45
```

It looks like it could be a bug, but I'm not sure

**Comment by digizeph** [comment]
@mbUSC Thanks for the bug report! I can confirm this issue. Will get on fixing it soon.

**Comment by digizeph** [comment]
@mbUSC The issue should be resolved in version `v0.10.4`. See #157 for more details of this fix.

The `bgpkit-parser` cli and library have both been updated. Please let me know if that resolves your issue. Thanks!

**Comment by mbUSC** [comment]
@digizeph Yes, that resolved the issue!
Thank you!
